pkg/types/aws/defaults/platform.go

```diff
@@ -72,3 +78,14 @@ func InstanceTypes(region string, arch types.Architecture, topology configv1.Top
 		}
 	}
 }
+
+// SkippedZones returns the list of supported zones.
+func SkippedZones(region string, zones []string) []string {
+	skipZones, ok := skippedZonesByRegion[region]
+	if (!ok) || (len(skipZones) == 0) {
+		return zones
+	}
+	zoneSet := sets.New[string](zones...)
+	skipSet := sets.New[string](skipZones...)
+	return sets.List(zoneSet.Difference(skipSet.Insert(skipZones...)))
```

Comment:
If using a list for the skipped zones:
```suggestion
	return sets.List(sets.New(zones...).Difference(sets.New(skippedZones...)))
```